dpnp/dpnp_iface_arraycreation.py

```diff
@@ -2286,17 +2457,67 @@ def linspace(
     )
 
 
-def loadtxt(fname, **kwargs):
+def loadtxt(
+    fname,
+    dtype=float,
+    like=None,
+    device=None,
+    usm_type=None,
```

Comment:
this change or modification to the document

```suggestion
    usm_type="device",
```